sensors/src/loadbalancer_sensor.py

```diff
@@ -0,0 +1,167 @@
+import subprocess
+import logging
+import requests
+from st2reactor.sensor.base import Sensor
+from st2reactor.container.sensor_wrapper import SensorService
+from amphorae import get_amphorae
+
+
+# pylint: disable=abstract-method
+class LoadbalancerSensor(Sensor):
+    def __init__(self, sensor_service: SensorService, config):
+        # pylint: disable=super-with-arguments
+        super(LoadbalancerSensor, self).__init__(
+            sensor_service=sensor_service, config=config
+        )
+        self.sensor_service: SensorService = sensor_service
+        self._logger = self.sensor_service.get_logger(name=self.__class__.__name__)
+
+    # pylint: disable=inconsistent-return-statements
+    def run(
+        self,
+        cloud_account: str = "dev-admin",
+    ):
+        """
+        Action to check loadbalancer and amphorae status
+        output suitable to be passed to create_tickets
+        """
+
+        amphorae = get_amphorae(cloud_account)
+
+        amph_json = self._check_amphora_status(amphorae)
+        # pylint: disable=line-too-long
+        output = {
+            "title": "{p[title_text]}",
+            "body": "The loadbalance ping test result was: {p[lb_status]}\nThe status of the amphora was: {p[amp_status]}\nThe amphora id is: {p[amp_id]}\nThe loadbalancer id is: {p[lb_id]}",
+            "server_list": [],
+        }
+        if amphorae.status_code != 200:
+            # Notes problem with accessing api if anything other than 403 or 200 returned
+            logging.critical("We encountered a problem accessing the API")
+            logging.critical("The status code was: %s ", str(amphorae.status_code))
+            logging.critical("The JSON response was: \n %s", str(amph_json))
+            return False
+
+        # Gets list of amphorae and iterates through it to check the loadbalancer and amphora status.
+        for i in amph_json["amphorae"]:
+            status = self._check_status(i)
+            ping_result = self._ping_lb(i["lb_network_ip"])
+            # This section builds out the ticket for each one with an error
+            if status[0] == "error" or ping_result == "error":
+                if status[0].lower() == "error" and ping_result.lower() == "error":
```

Comment:
I would simplify these conditions since the latter implies the former and humans struggle with multiple conditions in a row (it tends to be bug-prone). 
Simply use a chain of ifs (not elifs as it's not Pythonic, since people have to figure out if the previous condition affected the future one)

```python
if status[0].lower() == "error" and ping_result.lower() == "error":
   ...
if status[0].lower() != "error" and ping_result.lower() == "error":
  ...
if status[0].lower() == "error" and ping_result.lower() != "error":
  ...
if status[0].lower() != "error" and ping_result.lower() != "error":
   print("Everything is fine")
```

It also makes it easier to see we've handled 2^2 = 4 combos as there's 4 if branches.